### DISCUSSION TITLE
Garage & Lock mechanism

### DISCUSSION THREAD

**Discussion by iondarie** [HomeSpan/HomeSpan#102] [Q&A]
Hello,

I stumbled upon HomeSpan, but unfortunately my knowledge about API, Arduino and Programming seems to bee limited.

I would like to pay for someone to get my Garage and Lock mechanism to work with HomeSpan.

The garage has 1 relay for open/close and 2 reed sensors one for fully open and one for fully closed.

The lock mechanism has 1 relay for open and one reed sensor for open.

If anyone is interested please let me know.

Thank you

**Comment by danilkorotkov** [comment]
What about soldering?

**Reply by danilkorotkov** [reply]
![DF6DD16A-4559-4102-BD86-A3083E2B3D69](https://user-images.githubusercontent.com/25537053/123505565-83233e80-d668-11eb-8aca-3175b5803c05.jpeg)
2 limit sensors, open close and stop outputs and lock relay. Optosensors are also implemented

**Reply by danilkorotkov** [reply]
![5D4EB031-D54B-47FF-8CA9-6AE37F4919EB](https://user-images.githubusercontent.com/25537053/123505677-22483600-d669-11eb-85e3-8bb8c3ad8542.jpeg)

**Reply by iondarie** [reply]
I have no problem with soldering, i just do not know that to code for the gpio sensors and relays

**Reply by danilkorotkov** [reply]
https://github.com/danilkorotkov/HomeSpanGate
